eisbuk-admin/src/components/slots/SlotListByDay/SlotsDay.js

```diff
@@ -50,6 +55,9 @@ const SlotsDay = ({
   const copiedWeekSlots = copiedWeek.slots
     ? copiedWeek.slots.map((slot) => slot.id)
     : [];
+
+  const checkSelected = (id) => copiedWeekSlots.includes(id);
+  const canClickSlots = enableEdit && copiedWeekSlots.length > 0;
```

Comment:
I believe we should check on one more condition here: that the week currently displayed is the one currently in the clipboard.

With the current logic the user might select slots from two different weeks, and get surprising results on paste.

Other than this, I like the idea that, after deselecting all slots, the user needs to press the copy button again to get into "select mode".